### DISCUSSION TITLE
Import (and sync) Plex music library details

### DISCUSSION THREAD

**Discussion by arsaboo** [beetbox/beets#4077] [Ideas]
It would be great to import music library stats directly from Plex. I use Plex for my music needs and have created ratings and playlists. However, I like the idea of using Beets for managing the entire library. Thus, it would be great to import all the library details, like play count, ratings, playlists, etc. from Plex. 

There are several other plugins that import stats (e.g., LastImport), so I am assuming this is something that Beets would want to do. In addition, we already have the basic infrastructure in terms of Plex connectivity.

**Comment by sampsyo** [comment]
Seems perfectly reasonable; happy to have this tracked as a feature request.

**Reply by arsaboo** [reply]
Thanks! By feature request, do you mean that I should file it somewhere else or this is fine?

**Reply by sampsyo** [reply]
A proper GitHub Issue is the thing to do. It may be possible to convert a discussion thread into an issue, but otherwise we'll need a separate one.

**Reply by arsaboo** [reply]
Created an issue https://github.com/beetbox/beets/issues/4083

I guess, you will have to add the feature request label.
